guides/common/modules/ref_permissions-required-for-provisioning.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: REFERENCE
+
+[id="permissions-required-for-provisioning_{context}"]
+= Permissions Required for Provisioning
+
+The following table lists the permissions a non-admin user requires to provision hosts.
```

Comment:
Provision with what? Baremetal, registration (technically not provisioning, but still), or any other provisioning method, like azure?

Reply:
Thanks for taking a look, sorry for the late reply.

I actually don't know which use case this should cover. I'll try to find out from the reporter of this bug.

Reply:
The request is to update permissions for these methods:

* [Provisioning Virtual Machines on KVM (libvirt)](https://theforeman-foreman-documentation-preview-pr-2292.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Provisioning_Virtual_Machines_on_KVM_provisioning)
* [Provisioning Virtual Machines on OpenShift Virtualization](https://theforeman-foreman-documentation-preview-pr-2292.surge.sh/nightly/Provisioning_Hosts/index-satellite.html#Provisioning_Virtual_Machines_Kubevirt_provisioning)